src/addon/chrome/chromeFiles/locale/en_US/locale.dtd

```diff
@@ -97,7 +97,7 @@
 <!ENTITY options.debug.request "Client to Server communication (requests)">
 <!ENTITY options.debug.response "Server to Client communication (responses)">
 <!ENTITY options.debug.exceptions "Exceptions and State Machine Information">
-<!ENTITY options.debug.stream "Raw Dump/ Dump Byte Stream">
+<!ENTITY options.debug.stream "Raw Dump/Dump Byte Stream">
```

Comment:
This changes here should be a separate pull request. They are unrelated to "mailserver compatibility"

Reply:
done

Reply:
Not sure how to do separate pull requests until this pull request is merged.

Reply:
Reverted